db/schema.rb

```diff
@@ -89,7 +89,7 @@
 
   create_table "blood_sugars", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "blood_sugar_type", null: false
-    t.integer "blood_sugar_value", null: false
+    t.decimal "blood_sugar_value", null: false
```

Comment:
The migration says `numeric`. Does rails change that to `decimal` in the schema?

Reply:
Yup, rails is changing this to `decimal`. This should be fine, `numeric` and `decimal` are equivalent from Postgresql's perspective. 

> The types decimal and numeric are equivalent. Both types are part of the SQL standard.

[source](https://www.postgresql.org/docs/current/datatype-numeric.html)